### PR TITLE
[Doc] serve with triton

### PR DESCRIPTION
**Pull request by catchlove666**
# Description

Brief Description of the PR:
serve with triton

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -327,7 +327,32 @@ For more detail, please refer to the shell script `./tools/config_tfserving.sh`.
...
+```
+docker pull nvcr.io/nvidia/tritonserver:22.05-py3
+
+export TFRA_BRANCH="master"
+git clone -b $TFRA_BRANCH https://github.com/tensorflow/recommenders-addons.git
+cd recommenders-addons
+
+python configure.py
+bazel build //tensorflow_recommenders_addons/dynamic_embedding/core:_cuckoo_hashtable_ops.so ##bazel 5.1.1 is well tested
+cp bazel-bin/tensorflow_recommenders_addons/dynamic_embedding/core/_cuckoo_hashtable_ops.so /lib
```
Comment (rhdong) [inline README.md:350]: I guess the `/lib` is only a directory for saving the so files temporarily. maybe you can use an env variable for this. That will make readers know the directory can be customized by themselves according to their context.

File: README.md [hunk 2]
```diff
@@ -327,7 +327,32 @@ For more detail, please refer to the shell script `./tools/config_tfserving.sh`.
...
+### With Triton
+When building the custom operations shared library it is important to
+use the same version of TensorFlow as is being used in Triton. You can
+find the TensorFlow version in the [Triton Release
+Notes](https://docs.nvidia.com/deeplearning/triton-inference-server/release-notes/index.html). A
+simple way to ensure you are using the correct version of TensorFlow
+is to use the [NGC TensorFlow
+container](https://ngc.nvidia.com/catalog/containers/nvidia:tensorflow)
+corresponding to the Triton container. For example, if you are using
+the 23.05 version of Triton, use the 23.05 version of the TensorFlow
```
Comment (rhdong) [inline README.md:339]: Can we have a matrix indicating the mapping relationship between the NGC TF and TF/TFRA versions?

Comment (catchlove666) [reply README.md:339]: the versions relationship can be seen in [Triton Release
Notes](https://docs.nvidia.com/deeplearning/triton-inference-server/release-notes/index.html)

**Review by rhdong — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -327,7 +327,38 @@ For more detail, please refer to the shell script `./tools/config_tfserving.sh`.
...
+
+python configure.py
+bazel build //tensorflow_recommenders_addons/dynamic_embedding/core:_cuckoo_hashtable_ops.so ##bazel 5.1.1 is well tested
+mkdir /tmp/so
+#you can also use the so file from pip install package file from "(PYTHONPATH)/site-packages/tensorflow_recommenders_addons/dynamic_embedding/core/_cuckoo_hashtable_ops.so"
+cp bazel-bin/tensorflow_recommenders_addons/dynamic_embedding/core/_cuckoo_hashtable_ops.so /tmp/so
+
+#tfra saved_model directory "/models/model_repository"
+docker run --net=host -v /models/model_repository:/models nvcr.io/nvidia/tritonserver:22.05-py3 bash -c \
+  "export LD_LIBRARY_PATH=/opt/tritonserver/backends/tensorflow2:$LD_LIBRARY_PATH && export LD_PRELOAD="/tmp/so/_cuckoo_hashtable_ops.so:${LD_PRELOAD}" && tritonserver --model-repository=/models/ --backend-config=tensorflow,version=2"
```
Comment (MoFHeka) [inline README.md:356]: `LD_PRELOAD=/lib/_cuckoo_hashtable_ops.so:/lib/_redis_table_ops.so tritonserver --model-repository=/models/ --strict-model-config=false --backend-config=tensorflow,version=2`
is enough.

**Review by MoFHeka — COMMENTED** [review]
Should markdown doc use code style for shell command? Like this:
'''bash
docker run xxxx
'''

**Pull request closed without merging**